smoketest/make-bindings.sh

```diff
@@ -15,13 +15,24 @@ command -v subxt || cargo install subxt-cli \
     --git https://github.com/paritytech/subxt.git \
     --tag v0.37.0
 
-if ! lsof -Pi :11144 -sTCP:LISTEN -t >/dev/null; then
-    echo "substrate nodes not running, please start with the e2e setup and rerun this script"
-    exit 1
+eth_network="${ETH_NETWORK:-localhost}"
+polkadot_network="${POLKADOT_NETWORK:-localhost}"
+
+if [ "$polkadot_network" == "westend" ]; then
+  # Todo: There is no penpal nodes on westend yet
```

Comment:
Doesn't look like Penpal was deployed on Rococo either. Perhaps we can use another 3rd party parachain? Unfortunately it doesn't look like there are any other parachains besides the system chains. :/

Reply:
Yeah, it's not available on testnet. I think we can focus on AH first and test non-system parachain transfers later.

Reply:
Sounds good!